src/main/java/game/Gamestate.java

```diff
@@ -72,14 +72,10 @@ public void update(final List<String> input) {
 			highscoreScreen(input);
 			DisplayText.highscoreScreen(Game.getInstance().getHighscore());
 			break;
-		case STATE_PAUSE_SCREEN:
+		default:
```

Comment:
hier wordt hetzelfde gedaan, waarom maak je de default een van de states ipv dat de code een fout meldt in de logger en dan de state corrigeert?